pages/mentor/22-contributor_mentor_facilitation.md

```diff
@@ -2,7 +2,7 @@
 title: GSoC contributor - Mentor Facilitation
 section: Org Admin
 order: 22
-permalink: mentor/GSoC contributor-mentor-facilitation.html
```

Comment:
Ooops.  This is clearly wrong.

I'm concerned about renaming the actual file as it will break any existing deep links.  Maybe leave the file as student but keep the title fixes?

Reply:
@rspier, I've updated and force-pushed, let me know what you think.